### PR TITLE
Doc: Update deprecation notices to standard

### PR DESCRIPTION
**Pull request by karenzone**
Brings older and recent deprecation notices up to standard

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -134,29 +134,24 @@ and no codec for the request's content-type is found
...
-
 [id="plugins-{type}s-{plugin}-ecs_compatibility"]
 ===== `ecs_compatibility`
 
   * Value type is <<string,string>>
   * Supported values are:
-    ** `disabled`: unstructured connection metadata added at root level
-    ** `v1`,`v8`: headers added under `[@metadata][http][header]`. Some are copied to structured ECS fields `http`, `url`, `user_agent` and `host`
+  ** `disabled`: unstructured connection metadata added at root level
+  ** `v1`,`v8`: headers added under `[@metadata][http][header]`. Some are copied to structured ECS fields `http`, `url`, `user_agent` and `host`
```
Comment (karenzone) [inline docs/index.asciidoc:149]: Updated to asciidoc formatting standard

File: docs/index.asciidoc [hunk 2]
```diff
@@ -219,28 +214,22 @@ The host or ip to bind
...
 ===== `keystore_password`
+deprecated[3.1.0, Use <<plugins-{type}s-{plugin}-ssl_certificate>> and <<plugins-{type}s-{plugin}-ssl_key>> instead]
 
   * Value type is <<password,password>>
   * There is no default value for this setting.
-  * This option is deprecated
 
 Set the truststore password
 
-Note: This option is deprecated and it will be removed in the next major version of Logstash.
```
Comment (karenzone) [inline docs/index.asciidoc:241]: Isn't this info emitted in the deprecation notice log? I think we can remove it from here (and for other deprecated options).

**Comment by karenzone** [comment]
Hi @kares. I was left off of a recent doc review and I didn't get a chance to make suggestions/corrections. LMKWYT

**Review by kares — APPROVED** [review]
Thanks Karen, wasn't aware that that there's a formalized way of marking deprecated flags.
Just followed blindly what existed in the docs, going to circle you in for review next time (which might be sooner than you expect :stuck_out_tongue_winking_eye: )

**Pull request merged**